From a18ab708f0525334e123f64a039b6e44fccc64d7 Mon Sep 17 00:00:00 2001
From: Matthieu CHARETTE <matthieu.charette@gmail.com>
Date: Sat, 14 Sep 2024 16:36:13 +0200
Subject: [PATCH] Add HUAWEI MateBook D 15 AMD ACCEL properties

(cherry picked from commit 8ee3d4df80369eee1210dcec680a03994c08a217)

Resolves: RHEL-49810
---
 hwdb.d/60-sensor.hwdb | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index d88aea4928..3e32c327e6 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -607,6 +607,15 @@ sensor:modalias:i2c:bmc150_accel:dmi:*:svnHewlett-Packard:pnHPPavilionx2Detachab
 sensor:modalias:i2c:bmc150_accel:dmi:*:svnHewlett-Packard:pnHPProTablet408:*:rn8048:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; -1, 0, 0; 0, 0, 1
 
+#########################################
+# HUAWEI
+#########################################
+
+# HUAWEI MateBook D 15 AMD
+sensor:modalias:acpi:SMO8840*:dmi:*:svnHUAWEI:pnBOHK-WAX9X:*
+ ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
+ ACCEL_LOCATION=base
+
 #########################################
 # I.T.Works
 #########################################
